django_airavata/apps/admin/static/django_airavata_admin/src/components/gatewayprofile/StoragePreferenceEditor.vue

```diff
@@ -38,6 +41,21 @@ export default {
       required: true
     }
   },
+  methods: {
+    handleTypedInput() {
+      const evt = window.event;
```

Comment:
The event is passed as the first parameter to event handlers in Vue. Please use that instead of `window.event` which is deprecated.

Reply:
See also https://vuejs.org/v2/guide/events.html#Method-Event-Handlers